assets/styles/style.css

```diff
@@ -19,7 +19,7 @@ body {
 }
 
 #description {
-  text-align: center
+  text-align: center;
 }
```

Comment:
Pay attention to the font-weight and if you realize well you will see that there are margin top, right, bottom, and left lacking. Review that, please.